docs/v2/spec/2018_Modified_LAR_Code_Sheet.csv

```diff
@@ -0,0 +1,246 @@
+51,"Co-Applicant or Co-Borrower, Name and Version of Credit Scoring Model",9,Not applicable ,
+51,"Co-Applicant or Co-Borrower, Name and Version of Credit Scoring Model",10,No co-applicant,
+51,"Co-Applicant or Co-Borrower, Name and Version of Credit Scoring Model",1111,Exempt,
+52,Reason for Denial,1,Debt-to-income ratio,
+52,Reason for Denial,2,Employment history,
+52,Reason for Denial,3,Credit history,
+52,Reason for Denial,4,Collateral,
+52,Reason for Denial,5,"Insufficient cash (downpayment, closing costs)",
+52,Reason for Denial,6,Unverifiable information,
+52,Reason for Denial,7,Credit application incomplete,
+52,Reason for Denial,8,Mortgage insruance denied,
+52,Reason for Denial,9,Other,
+52,Reason for Denial,10,Not applicable. ,Note: Code 10 is only valid for Reason for Denial: 1
+52,Reason for Denial,1111,Exempt. ,Note: Code 1111 is only valid for Reason for Denial: 1. 
+63,Debt-to-Income Ratio,Exact values between 36% and 50%,Published with modification. ,
+63,Debt-to-Income Ratio,<20%,Debt-to-income ratio is less than 20%,
+63,Debt-to-Income Ratio,20%-<30%,Debt-to-income ratio is between 20% and is less than 30%,
+63,Debt-to-Income Ratio,30%-<36%,Debt-to-income ratio is between 30% and is less than 36%,
+63,Debt-to-Income Ratio,50%-60%,Debt-to-income ratio is between 50% and 60%,
+63,Debt-to-Income Ratio,>60%,Debt-to-income ratio is greater than 60%,
+63,Debt-to-Income Ratio,NA,Not applicable,
+63,Debt-to-Income Ratio,Exempt ,Exemption taken ,
+67,Balloon Payment,1,Balloon payment,
+67,Balloon Payment,2,No balloon payment,
+67,Balloon Payment,1111,Exempt,
+68,Interest-Only Payments,1,Interest-only payments,
+68,Interest-Only Payments,2,No interest-only payments,
+68,Interest-Only Payments,1111,Exempt,
+69,Negative Amortization,1,Negative amortization,
+69,Negative Amortization,2,No negative amortization,
+69,Negative Amortization,1111,Exempt,
+70,Other Non-Amortizing Features,1,Other non-fully amortizing features,
+70,Other Non-Amortizing Features,2,No other non-fully amortizing features,
+70,Other Non-Amortizing Features,1111,Exempt,
+72,Manufactured Home Secured Property Type,1,Manufactured home and land,
+72,Manufactured Home Secured Property Type,2,Manufactured home and not land,
+72,Manufactured Home Secured Property Type,3,Not applicable ,
+72,Manufactured Home Secured Property Type,1111,Exempt,
+73,Manufactured Home Land Property Interest,1,Direct ownership,
+73,Manufactured Home Land Property Interest,2,Indirect ownership,
+73,Manufactured Home Land Property Interest,3,Paid leasehold,
+73,Manufactured Home Land Property Interest,4,Unpaid leasehold,
+73,Manufactured Home Land Property Interest,5,Not applicable ,
+73,Manufactured Home Land Property Interest,1111,Exempt,
+74,Total Units,1,1-Unit,
+74,Total Units,2,2-Unit,
+74,Total Units,3,3-Unit,
+74,Total Units,4,4-Unit,
+74,Total Units,24-May,Total units between 5 and 24,

```

Comment:
I think you got a date conversion here by accident.

Reply:
Thanks for the catch! A change was made in the last commit.